packages/blob-storage-manager/src/env.ts

```diff
@@ -1,59 +1,24 @@
-import { z } from "zod";
-
-function transformToBoolean(varName: string) {
-  return (arg: string, ctx: z.RefinementCtx): boolean => {
-    const arg_ = arg.toLowerCase();
-    if (arg_ !== "true" && arg_ !== "false") {
-      ctx.addIssue({
-        code: z.ZodIssueCode.custom,
-        message: `${varName} must be either true or false`,
-      });
-
-      return z.NEVER;
-    }
-
-    return arg_ === "true";
-  };
-}
-
-const envSchema = z.object({
-  BEE_DEBUG_ENDPOINT: z.string().url().optional(),
-  BEE_ENDPOINT: z.string().url().optional(),
-  CHAIN_ID: z
-    .string()
-    .min(1)
-    .default("7011893055")
-    .transform((value, ctx) => {
-      const chainId = parseInt(value, 10);
-
-      if (isNaN(chainId) || chainId <= 0) {
-        ctx.addIssue({
-          code: z.ZodIssueCode.custom,
-          message: `CHAIN_ID is invalid: ${chainId}`,
-        });
-
-        return z.NEVER;
-      }
-
-      return chainId;
-    }),
-  GOOGLE_STORAGE_BUCKET_NAME: z.string().optional(),
-  GOOGLE_STORAGE_PROJECT_ID: z.string().optional(),
-  GOOGLE_SERVICE_KEY: z.string().optional(),
-  GOOGLE_STORAGE_API_ENDPOINT: z.string().url().optional(),
-
-  GOOGLE_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("GOOGLE_STORAGE_ENABLED")),
-  POSTGRES_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("POSTGRES_STORAGE_ENABLED")),
-  SWARM_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("SWARM_STORAGE_ENABLED")),
+import {
+  z,
+  chainIdSchema,
+  booleanSchema,
+  createEnvSchema,
+} from "@blobscan/zod";
+
+const envSchema = createEnvSchema({
+  BEE_DEBUG_ENDPOINT: { schema: z.string().url(), optional: true },
+  BEE_ENDPOINT: { schema: z.string().url(), optional: true },
+  CHAIN_ID: { schema: chainIdSchema(), default: 7011893055 },
+  GOOGLE_STORAGE_BUCKET_NAME: { optional: true },
+  GOOGLE_STORAGE_PROJECT_ID: { optional: true },
+  GOOGLE_SERVICE_KEY: { optional: true },
+  GOOGLE_STORAGE_API_ENDPOINT: {
+    schema: z.string().url(),
+    optional: true,
+  },
+  GOOGLE_STORAGE_ENABLED: { schema: booleanSchema(), default: false },
+  POSTGRES_STORAGE_ENABLED: { schema: booleanSchema(), default: false },
```

Comment:
Yes, this was part of the discussions I had with @PabloCastellano. We believed it would be better for the user to explicitly set the storages they want to use.

Reply:
> Yes, this was part of the discussions I had with @PabloCastellano. We believed it would be better for the user to explicitly set the storages they want to use.

But for postgres, true is a saner default.

Reply:
Oops, I got that backwards. I'll set this one to `true` then